homeassistant/components/lacrosse_view/manifest.json

```diff
@@ -5,5 +5,6 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/lacrosse_view",
   "iot_class": "cloud_polling",
-  "requirements": ["lacrosse-view==0.0.9"]
+  "loggers": ["lacrosse_view"],
+  "requirements": ["lacrosse-view==1.0.1"]
```

Comment:
Library bump should be a standalone PR (maybe with adjustment to loggers in manifest)

Reply:
All the library bump does is add better errors and debug logs. Should it be separate regardless?

Reply:
Yes